controllers/platformnetwork_controller.go

```diff
@@ -317,6 +420,18 @@ func (r *PlatformNetworkReconciler) ReconcileAddressPool(client *gophercloud.Ser
 	} else {
 		if pool == nil {
 			pool, err = r.ReconcileNewAddressPool(client, instance)
+		} else if instance.Spec.Type == cloudManager.OAMNetworkType {
+			oamNetworkList, err := oamNetworks.ListNetworks(client)
+			if err != nil {
+				err = perrors.Wrapf(err, "failed to get oam network")
+				return err
+			}
+			oamNetwork := &oamNetworkList[0]
+			err = r.ReconcileUpdatedOAMNetwork(client, instance, oamNetwork)
```

Comment:
Does it mean we are supporting OAM network reconfiguration here?

Reply:
Yes, that's a part of Hugo's fix. OAM network reconfiguration is supported in the current release, I would prefer to pick it up